server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -351,10 +376,17 @@ async fn expunge_message_content(
 
 pub fn router() -> ApiRouter<AppState> {
     let tag = openapi_tag("Message");
+
+    fn document_create_message(op: TransformOperation) -> TransformOperation {
+        op.response::<202, Json<MessageOut>>()
+            .response::<200, ()>()
```

Comment:
```suggestion
            .response::<200, Json<()>>()
```
No?

Reply:
That depends on what you want. `()` is a response with no body at all and no `Content-Type` either. `Json<()>` will have `Content-Type: application/json`, but I'm not sure what `()` is in JSON, it doesn't really have a meaningful equivalent. I went with `()` because that's really _nothing at all_.

We can also do `Json<EmptyResponse>` which I think we currently serialize as `{}`.

Reply:
Ugh, `()` is serialized as `null`: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=a77a840624e3b8377952dbfc1982ff81

So, you probably don't want `Json<()>`

Reply:
Json<EmptyResponse> sounds right

Reply:
I guess by that you mean `Json<EmptyResponse>` right?

Reply:
Changed response to `Json<EmptyResponse>`